.github/workflows/build-rhel-bootc.yaml

```diff
@@ -0,0 +1,100 @@
+name: "Build RHEL Bootc Agent Bootstrap Image" 
+on:
+  workflow_dispatch
+env:
+  REGISTRY: quay.io
+  REGISTRY_USER: ${{ github.actor }}
+
+jobs:
+  build-and-push-rhel:
+    runs-on: ubuntu-latest
+
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Modify RHEL Containerfile
+        run: |
+          pushd bootc-agent-images/rhel
+          echo "${{ secrets.FLIGHTCTL_RSA_PUB }}" > flightctl_rsa.pub
+          sed -i -e 's/^# COPY flightctl_rsa.pub \/usr\/etc-system\/root.keys/COPY flightctl_rsa.pub \/usr\/etc-system\/root.keys/' \
+                 -e 's/^# RUN touch \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/RUN touch \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/' \
+                 -e 's/^#     mkdir -p \/usr\/etc-system\/;/    mkdir -p \/usr\/etc-system\/;/' \
+                 -e 's/^#     echo '\''AuthorizedKeysFile \/usr\/etc-system\/%u.keys'\'' >> \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/    echo '\''AuthorizedKeysFile \/usr\/etc-system\/%u.keys'\'' >> \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/' \
+                 -e 's/^#     chmod 0600 \/usr\/etc-system\/root.keys/    chmod 0600 \/usr\/etc-system\/root.keys/' \
+                 -e 's/^# VOLUME \/var\/roothome/VOLUME \/var\/roothome/' Containerfile
+          echo "${{ secrets.CA_CRT }}" > ca.crt
+          echo "${{ secrets.CLIENT_ENROLLMENT_CRT }}" > client-enrollment.crt
+          echo "${{ secrets.CLIENT_ENROLLMENT_KEY }}" > client-enrollment.key
+          popd
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: network=host
+
+      - name: Login to registry.redhat.io
+        uses: docker/login-action@v3
+        with:
+          registry: registry.redhat.io
+          username: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}
+          password: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}
+
+      - name: Login to registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}
+          password: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}
+
+      - name: Build RHEL image
+        uses: docker/build-push-action@v5
+        with:
+          context: bootc-agent-images/rhel
+          file: bootc-agent-images/rhel/Containerfile
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: localhost:5000/user/flightctl-agent-rhel:bootstrap
```

Comment:
Please, let's change the bootstrap tag, for bootstrap-experimental. 

I have replicated the periodic pipelines for RHEL, and this new pipeline you are developing will need to be tagged differently.